examples-features/simple_ble.rs

```diff
@@ -7,6 +7,11 @@ use libtock::simple_ble::BleAdvertisingDriver;
 use libtock::timer::Duration;
 use serde::Serialize;
 
+/// Dummy buffer that causes the linker to reserve enough space for the stack.
+#[no_mangle]
+#[link_section = ".stack_buffer"]
+pub static mut STACK_MEMORY: [u8; 0x800] = [0; 0x800];
```

Comment:
Given that all examples have the same stack size it makes sense to me to put the value in an enviroment variable, read this in a procedural macro, and hide the macro invocation in `libtock-rs`. As "real" projects (non-examples) only have ony binary per crate with a fixed stack size, this should give us enough flexibility. Still, I am not convinced that creating this symbol in the code instead of templating the layout files is a good solution.

Reply:
I don't think they should all have the same size. I started by minimising changes to avoid regressions. The next step I think should be to shrink all these sizes down.